subjects/devops/left/README.md

```diff
@@ -0,0 +1,29 @@
+## left
+
+### Instructions
+
+Create a file `left.sh` that will pass the content of a file `facts` to a command that will return the number of lines in that file.
+
+### Usage
+
+```console
+$./left.sh
+21
+```
+
+### Hints
+
+To pass the content of a file to a command in a shell, you can use the "<" operator followed by the name of the file. For example, to pass the content of a file named "input.txt" to the cat command, you can use the following command:
+
+`cat < input.txt`
```

Comment:
I feel the example command should be something like `< input.txt cat` instead of the one presented. The result is the same but it shows better `<` metacharacter's effect